CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 - The image viewBox is now properly translated
 - `vello_svg::render_tree_with` no longer takes a transform parameter. This is to make it consistent with the documentation and `vello_svg::render_tree`.
 
+## 0.1.1 (Yanked)
```

Comment:
```suggestion
```

I don't think this really adds anything, and the first actual release to contain these changes will be 0.2.0, I think